packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -19,15 +19,6 @@
 
 This library provides constructs for Node.js Lambda functions.
 
-To use this module, you will need to add a dependency on `parcel-bundler` in your
```

Comment:
Explain that docker is now a dependency? What about leaving the choice up to the user?

Reply:
Done. I don't think having the option to use the npm package is needed. Docker is already used for assets.

Reply:
I am okay with always requiring docker to keep this simple and extensible.